osf/models/schema_response_block.py

```diff
@@ -22,28 +23,33 @@ class SchemaResponseBlock(ObjectIDMixin, BaseModel):
     source_schema_response = models.ForeignKey(
         'osf.SchemaResponse',
         null=False,
-        related_name='updated_response_blocks'
+        related_name='updated_response_blocks',
+        on_delete=models.CASCADE,
     )
-    # The RegistrationSchemaBlock that defines the question being answered
-    source_schema_block = models.ForeignKey('osf.RegistrationSchemaBlock', null=False)
-
-    # Should match source_schema_block.registration_response_key
+    # Must align with the `registration_response_key from some schema_block on the
```

Comment:
```suggestion
    # Must match the `registration_response_key` for some schema_block on the
```

Reply:
Are we doing any validation when this is set to ensure it references a block correctly? 
Uniqueness should be ensured with the `unique_together`, so that's not  a concern

Reply:
It's still being passed a SchemaBlock on construction and not just a key, so the interface should handle this

Reply:
(to be clear, the case where the interface doesn't handle this is purely if a block gets deleted, which will raise errors the next time we try to assign a value to the key)